kura/pom.xml

```diff
@@ -69,7 +69,6 @@
         <module>org.eclipse.kura.wire.provider</module>
         <module>features</module>
         <module>emulator</module>
-        <module>test</module>
```

Comment:
I don't like that this update involves removing tests by default.

If you do this, please add another profile, that is activated by default:
```
        <profile>
            <id>tests</id>
            <activation>
                <property>
                    <name>!skipTests</name>
                </property>
            </activation>
            <modules>
                <module>test</module>
            </modules>
        </profile>
```

Then, use the following script for the first stage:
`- $M2_HOME/bin/mvn -B -f kura/pom.xml clean install -DskipTests`

This way the tests would still start when building manually.

Similar should probably be done for the `org.eclipse.kura-p2` module as it now builds twice.